packages/cursorless-everywhere-talon/scripts/populate-dist.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+mkdir dist
+
+cp -r out/. dist
+
+cp -r ../../cursorless-everywhere-talon/. dist
```

Comment:
would be easier to read if you found a way to use our `CURSORLESS_REPOSITORY_ROOT` env var. If awkward to get access to that, maybe just define it at the top so it's clearer what's happening here. For a while I thought `cursorless-everywhere-talon` was referring the the parent directory of this script

Reply:
I do agree it's a bit confusing. I'm not sure the best way of doing that in bash. Can I just leave a comment instead?

Reply:
Just define a variable of that name to point to ../.. if it's hard to expose the proper one

Reply:
Sounds good

Reply:
update from meet-up: delete this file